api/src/main/java/org/openmrs/scheduler/timer/TimerSchedulerServiceImpl.java

```diff
@@ -542,5 +542,14 @@ public void scheduleIfNotRunning(TaskDefinition taskDef) {
 			}
 		}
 	}
+
+	/**
```

Comment:
Is there any reason why you did not add `@Transactional(readOnly = true)` like the rest of the getters?

Reply:
not really,just didn't notice it but i have added it